README_PODMAN.md

```diff
@@ -0,0 +1,44 @@
+# Using the dnsname plugin with Podman
+
+The purpose of the *dnsname* plugin is to allow containers to resolve each other by name.  The plugin adds each
+container's name to an instance of a dnsmasq server.  The plugin is enabled through adding it to a network's
+CNI configuration.  The containers will only be able to resolve each other if they are on the same CNI network.
+
+**Note**: This only works with non-rootless containers.
```

Comment:
The double negative is iffy to me.
I'd suggested `this only works with containers spawned by the root user` or something

Reply:
This does not work with rootless containers.